src/utils/Utils.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Utils functions
+ */
+export class Utils {
+    /**
+     * Filter records to be sent to SQS
+     * @param records
+     */
+    public static filterCertificateGenerationRecords(records: any[]) {
+        return records
+            .filter((record: any) => { // Filter by testStatus
+                return record.testStatus === "submitted";
+            })
+            .filter((record: any) => { // Filter by testResult (abandoned tests are not allowed)
+                return (record.testTypes.testResult === "pass" || record.testTypes.testResult === "fail" || record.testTypes.testResult === "prs");
```

Comment:
2 out of 3 conditions tested
![image](https://user-images.githubusercontent.com/50110006/71975977-241c0080-320d-11ea-8c93-048f35eeb64d.png)

Reply:
Done:
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/50991726/71979473-bcb67e80-3215-11ea-83cc-55b1bff9f750.png">